labs/kafka-avro/java/producer/src/main/java/clients/ShakespeareProducer.java

```diff
@@ -0,0 +1 @@
+// TODO: Add code here
```

Comment:
There should be more than just "Add code here". This file should contain:

`package clients;`
`import java.io.BufferedReader;`
`import java.io.File;`
`import java.io.FileNotFoundException;`
`import java.io.FileReader;`
`import java.io.IOException;`
`import java.util.Properties;`
`import org.apache.kafka.common.serialization.StringSerializer;`
`import org.apache.kafka.clients.producer.KafkaProducer;`
`import org.apache.kafka.clients.producer.ProducerRecord;`
`import org.apache.kafka.clients.producer.ProducerConfig;`
`public class ShakespeareProducer {`
`private final static String INPUT_PATH_NAME = "/datasets/shakespeare";`
`// here will be the Kafka Producer code`
`}`

Reply:
not addressed